test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/HtmlErrorTest/ThrowsErrorIfEndTextTagContainsTextAfterName.stree.txt

```diff
@@ -1,16 +1,16 @@
-Markup block - Gen<None> - 21 - (0:0,0)
-    Tag block - Gen<None> - 6 - (0:0,0)
-        Transition span - Gen<None> - [<text>] - SpanEditHandler;Accepts:None - (0:0,0) - Tokens:3
-            SyntaxKind.OpenAngle;[<];
-            SyntaxKind.Text;[text];
-            SyntaxKind.CloseAngle;[>];
-    Tag block - Gen<None> - 15 - (6:0,6)
-        Transition span - Gen<None> - [</text foo bar>] - SpanEditHandler;Accepts:Any - (6:0,6) - Tokens:8
-            SyntaxKind.OpenAngle;[<];
-            SyntaxKind.ForwardSlash;[/];
-            SyntaxKind.Text;[text];
-            SyntaxKind.Whitespace;[ ];
-            SyntaxKind.Text;[foo];
-            SyntaxKind.Whitespace;[ ];
-            SyntaxKind.Text;[bar];
-            SyntaxKind.CloseAngle;[>];
+MarkupBlock - [0..21)::21 - [<text></text foo bar>]
+    MarkupTagBlock - [0..6)::6 - [<text>]
+        MarkupTransition - [0..6)::6 - Gen<None> - SpanEditHandler;Accepts:None
+            OpenAngle;[<];
+            Text;[text];
+            CloseAngle;[>];
+    MarkupTagBlock - [6..21)::15 - [</text foo bar>]
+        MarkupTransition - [6..21)::15 - Gen<None> - SpanEditHandler;Accepts:Any
+            OpenAngle;[<];
+            ForwardSlash;[/];
+            Text;[text];
+            Whitespace;[ ];
+            Text;[foo];
+            Whitespace;[ ];
+            Text;[bar];
+            CloseAngle;[>];
```

Comment:
This seems a little odd - why are all of these additional tokens part of a transition? Is this just a recovery bug in the parser? Should we log this?

Reply:
This already throws an error `(1,9): Error RZ1023: "<text>" and "</text>" tags cannot contain attributes.`. The `*.diag.txt` file was unchanged.

Reply:
So why is it treated a transition with like 8 things in it?

Reply:
Spoke offline. This is a `MarkupTransition` as opposed to a `CSharpTransition`.